recipe/meta.yaml

```diff
@@ -18,23 +18,28 @@ requirements:
   host:
     - python
     - pip
-    - setuptools
-    - wheel
+    - setuptools >=64
+    - wheel >=0.37.1
```

Comment:
Pinning isn't needed for Python packages in `host`. Plain `setuptools` and `wheel` are enough because the latest version will be used in any case. What does matter is an upper bound when the upstream issues are reported. But it's not that case

Reply:
Here is the recipe standards for the host section https://github.com/anaconda-distribution/perseverance-skills/blob/main/sections/02_Package_building/02_Recipes/Recipe_standards.md#the-requirements-section